From ccb46d92d1fe0883818c800462813c32f1fe28aa Mon Sep 17 00:00:00 2001
From: Hyun Kwon <hyun.kwon@xilinx.com>
Date: Mon, 9 Feb 2015 10:14:05 -0800
Subject: [PATCH 498/793] drm: xilinx: Kconfig: Remove depend on ARCH_ZYNQ

The Xilinx DRM is not ZYNQ specific anymore. Remove the dependency
from Kconfig.

Signed-off-by: Hyun Kwon <hyun.kwon@xilinx.com>
Signed-off-by: Michal Simek <michal.simek@xilinx.com>
---
 drivers/gpu/drm/xilinx/Kconfig |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/drivers/gpu/drm/xilinx/Kconfig b/drivers/gpu/drm/xilinx/Kconfig
index df608d0..3339154 100644
--- a/drivers/gpu/drm/xilinx/Kconfig
+++ b/drivers/gpu/drm/xilinx/Kconfig
@@ -1,6 +1,6 @@
 config DRM_XILINX
 	tristate "Xilinx DRM"
-	depends on DRM && ARCH_ZYNQ && HAVE_CLK
+	depends on DRM && HAVE_CLK
 	select DRM_KMS_HELPER
 	select DRM_KMS_CMA_HELPER
 	select DRM_GEM_CMA_HELPER
-- 
1.7.5.4

